api/grpcserver/http_server.go

```diff
@@ -10,10 +10,19 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/rs/cors"
+	httpMetrics "github.com/slok/go-http-metrics/metrics"
+	metricsProm "github.com/slok/go-http-metrics/metrics/prometheus"
+	"github.com/slok/go-http-metrics/middleware"
+	"github.com/slok/go-http-metrics/middleware/std"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
+
+	"github.com/spacemeshos/go-spacemesh/metrics"
 )
 
+// recorder is a global recorder for http metrics.
+var recorder *httpMetrics.Recorder
```

Comment:
I wonder why is a global variable needed here? `StartService` is called once in the whole codebase. It can just live as a variable scoped in the method, WDYT?

Reply:
Fixed